test/frame.test.js

```diff
@@ -15,7 +15,10 @@ describe('Recurly.Frame', function () {
       publicKey: 'test',
       api: `//${window.location.host}/api`
     });
-    window.open = sinon.spy();
+
+    // Returns a spy so frame.bindWindowCloseListener does not cancel the tick
+    window.open = sinon.spy(() => sinon.spy());
```

Comment:
This is necessary because of this line of code https://github.com/recurly/recurly-js/blob/aac745c187f713bdfed4c739b10574f5471acd12/lib/recurly/frame.js#L103